### PR TITLE
record policy: remove Admin permissions

### PR DESCRIPTION
**Pull request by kpsherva**

### CODE REVIEW

File: run-tests.sh [hunk 1]
```diff
@@ -23,9 +23,9 @@ function cleanup() {
 trap cleanup EXIT
 
 
-python -m check_manifest
-python -m setup extract_messages --dry-run
-python -m sphinx.cmd.build -qnNW docs docs/_build/html
+#python -m check_manifest
```
Comment (ntarocco) [inline run-tests.sh:26]: oups

**Review by ntarocco — COMMENTED** [review]
Maybe change the commit msg?

File: run-tests.sh (hunk 1 above)
Comment (kpsherva) [reply run-tests.sh:26]: 👀

File: tests/test_record_permission_policy.py [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+    assert create_perm.excludes == {any_user}
+
+
+def test_update_record_policy(create_record):
+    """Test create record policy."""
+    record = create_record()
+    update_perm = RecordPermissionPolicy(action="update", record=record)
+
+    # only owners should be able to update
+    assert update_perm.needs == {UserNeed(1), UserNeed(2), UserNeed(3)}
```
Comment (zzacharo) [inline tests/test_record_permission_policy.py:40]: SHouldn't this be also valid for `superuser-access`?

**Review by zzacharo — APPROVED** [review]

**Review by zzacharo — APPROVED** [review]

**Review by ntarocco — APPROVED** [review]

**Pull request merged**